### DISCUSSION TITLE
Renaming Sections & Inner Sections

### DISCUSSION THREAD

**Discussion by MrMono** [elementor/elementor#18873] [Developers Community]
Sometimes when working on a page is difficult to select different containers and sections so the Navigator helps. I was wondering if it would be a good idea to be able to rename these sections a la Webflow. I find myself clicking different 'Sections' trying to find the one I want to edit. 

Thoughts? 🤔

![image](https://user-images.githubusercontent.com/25699960/172423218-5df73774-d49d-4573-9cbd-5898a19f8605.png)

**Comment by heroicP** [comment]
Good news! This feature already exists! Just double click the Inner Section or Section and it should let you name them.

**Reply by MrMono** [reply]
Thanks for the reply @heroicP 🙌I will give this a try.
